ksql-engine/src/main/java/io/confluent/ksql/ddl/commands/RegisterTopicCommand.java

```diff
@@ -78,12 +78,20 @@ private KsqlTopicSerDe extractTopicSerDe(
       case DataSource.JSON_SERDE_NAME:
         return new KsqlJsonTopicSerDe();
       case DataSource.DELIMITED_SERDE_NAME:
+        if (properties.containsKey(DdlConfig.VALUE_DELIMITER_PROPERTY)) {
```

Comment:
Rather than adding this if/else logic you could just use `properties.getOrDefault`, e.g.

```java
// In DdlConfig
public static final StringLiteral VALUE_DELIMITER_DEFAULT = new StringLiteral(",");

// Then you can write:
final String delimiter = StringUtil.cleanQuotes(properties.getOrDefault(
   DdlConfig.VALUE_DELIMITER_PROPERTY, 
   DdlConfig.VALUE_DELIMITER_DEFAULT).toString());
return new KsqlDelimitedTopicSerDe(delimiter);
```

I'd then make it so that the `KsqlDelimitedTopicSerDe` constructor always takes a String.